frontend/package.json

```diff
@@ -23,15 +23,19 @@
     "@mantine/nprogress": "^7.11.2",
     "@mantine/spotlight": "^7.11.2",
     "@mantine/tiptap": "^7.11.2",
+    "@mui/icons-material": "^6.0.0",
```

Comment:
Preferably uninstall MUI to keep the project clean + to avoid any importing issues